### ISSUE TITLE
Indoor Temperature Readings Override

### ISSUE THREAD

**Issue by bzumik1** [jourdant/esphome-lgap#8]
I am opening this issue to track the progress of this feature. It should allow you to override the room temperature. This will enable you to get rid of the LG thermostat and combine the temperature reading from multiple sensors for better accuracy.

This functionality is available as part of this integration [this repository](https://github.com/JanM321/esphome-lg-controller) but based on the information we have, it should be possible to add it using LGAP as well.

**Comment by MaikaiLife** [comment]
> I am opening this issue to track the progress of this feature. It should allow you to override the room temperature. This will enable you to get rid of the LG thermostat and combine the temperature reading from multiple sensors for better accuracy.
> 
> This functionality is available as part of this integration [this repository](https://github.com/JanM321/esphome-lg-controller) but based on the information we have, it should be possible to add it using LGAP as well.

@bzumik1 

Did you ever get this working, myself and a couple of others would like to keep this project going but it seems everyone initially involved has dropped off since about June of 2024.

**Comment by mystardious** [comment]
I don't think this is possible since what it does is change the reported room temperature by the LG Controller back to the ODU and AFAIK isn't supported.
